crates/pallet-executor/src/lib.rs

```diff
@@ -84,7 +84,7 @@ mod pallet {
     #[pallet::without_storage_info]
     pub struct Pallet<T>(_);
 
-    #[derive(TypeInfo, Encode, Decode, PalletError)]
+    #[derive(TypeInfo, Encode, Decode, PalletError, Debug)]
```

Comment:
The primary purpose of `RuntimeDebug` is to not use the regular full Debug impl under no_std in order to avoid bloating the final wasm blob from non critical code, instead you'll just have `<wasm:stripped>` for Debug under no_std. Do you want to force using `RuntimeDebug` by default? I don't have a strong preference here honstly.

Reply:
Hm, no strong preference either